app/components/reports/reports-filters.tsx

```diff
@@ -46,24 +47,31 @@ const FilterItems: React.FC<ReportFiltersProps> = ({
 					minStepsBetweenThumbs={18}
 				/>
 			</div>
-			<div className="w-full h-[1px] bg-vd-blue-900 my-6 md:my-10" />
+			<Separator className="bg-vd-blue-500 my-6 md:my-10" />
 			<div className="flex flex-col md:flex-row md:justify-between">
 				<div className="pl-4">
-					<h2 className="font-medium pb-2 md:pb-4">Story from</h2>
+					<h2 className="text-lg font-medium pb-2 md:pb-4">Story from</h2>
 					{outlets.map((outlet: string) => (
-						<div key={outlet} className="flex items-center gap-2 pb-1">
-							<Circle size={18} strokeWidth={1} />
-							<p className="text-xs">{outlet}</p>
+						<div key={outlet} className="flex items-center gap-2 pb-2">
+							<Checkbox className="h-6 w-6 rounded-full border-vd-blue-500 data-[state=checked]:bg-vd-blue-500 data-[state=checked]:text-vd-beige-100" />
```

Comment:
Let's not style these like radio buttons, but instead keep them "boxy" like:
![image](https://github.com/VoiceDeck/app/assets/19380973/eb97baf1-fff7-4ee6-9796-7b915592af22)